nb-extension/js/main.js

```diff
@@ -20,11 +20,25 @@ define([
         window.console[method] = window.console[method] || window.console.log;
     });
     
-    init({
-        namespace: Jupyter,
-        events: events,
-        WidgetManager: widgetManager.WidgetManager,
-        WidgetModel: ipywidget.WidgetModel
+    // setup package name; required for later use by jupyter-js-widgets
+    requirejs({
```

Comment:
So it seems that you can use require.config() to configure... but not shadow `require` in your callback function.

```
define([
    'require',
    'base/js/namespace',
    'base/js/events',
    'nbextensions/widgets/widgets/js/init',
    'nbextensions/widgets/widgets/js/widget'
], function(requirejs, Jupyter, events, widgetManager, ipywidget) {
    'use strict';
```